memfreeme/memfree
# Fix pagespeed accessibility issues 44

# Pull Request for Memfree

## Description

This pull request addresses accessibility issues for the Memfree site by implementing fixes based on a local Google Lighthouse audit. These changes improve the website's accessibility and alignment with WCAG standards, enhancing the overall user experience.

## Related Issue

- Fixes #44 - "Fix memfree.me pagespeed Accessibility issues"

## Changes Made

- [x] **Added aria-labels**: Included `aria-label` attributes to all buttons in the frontend directory to ensure buttons have accessible names.
- [x] **Improved link accessibility**: Updated links to have discernible names, resolving the "Links do not have a discernible name" issue.
- [x] **Enhanced color contrast**:
  - Changed sidebar background color on the homepage.
  - Increased font weight for the "UI" text on the sidebar.
  - Updated font weight and colors for "Attach" and "Index" labels and their icons below the homepage input field.

## How to Test

To test the changes:

1. Run a Google Lighthouse audit on the homepage to verify improved accessibility scores.
2. Confirm all buttons have accessible names and links are discernible.
3. Review color contrast changes to ensure they meet WCAG compliance.

## Screenshots (if applicable)
<img width="1440" alt="Screenshot 2024-10-27 at 21 12 07" src="https://github.com/user-attachments/assets/f46e402e-826e-4954-a2d5-6d35b0d8a78e">


Please add screenshots to showcase visual modifications.

## Checklist

- [x] My code is tested and works as expected.

## Additional Information

Any additional information helpful for the reviewers can be added here.
